Dockerfile

```diff
@@ -1,4 +1,4 @@
-# TODO: make this default to govuk-ruby once it's being pushed somewhere public
+FROM ruby:2.6.9
```

Comment:
Buggy output from [upgrade-ruby-version](https://github.com/alphagov/upgrade-ruby-version). Works for most apps, but Whitehall is a special case!